EasyPost/Utilities/Internal/Extensions/Dictionaries.cs

```diff
@@ -1,5 +1,7 @@
+using System;
```

Comment:
Thought: The amount of utility code this repo needs is astronomically too large for what the library actually does/is. I'd like us over time to be trimming this down, not adding more to it.

How can we do more with less? What patterns can we identify and refactor (without affecting user expectations and behavior under the hood) to remove dead or duplicated code? Something to think and improve on.

Reply:
I can start in the next PR by ripping out the three overloads for this method that I added that aren't being used :)